src/client/src/Pages/CreateJob.tsx

```diff
@@ -1,7 +1,179 @@
 import React from 'react';
+import { Formik, Form, Field} from "formik";
+import Button from "@material-ui/core/Button";
+import * as Yup from 'yup'; 
 
-function CreateJob() {
-    return <h1>TODO: Adds components for the create jobs page</h1>;
+import FormikField from "../Components/JobForm/FormikField";
+import FormikSelect, { FormikSelectItem } from "../Components/JobForm/FormikSelect";
+
+
+interface FormValues {
```

Comment:
I believe this form values do not quite correspond to the values expected by the backend. You can take a look at Mari's endpoint to create a job to see what data is expected to be sent from the client. Refer server/src/routes/job.ts